hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -29,10 +29,35 @@
 #include "hardware_interface/introspection.hpp"
 #include "hardware_interface/macros.hpp"
 
+namespace
+{
+std::string get_data_type_name(const hardware_interface::HandleDataType & data_type)
+{
+  switch (data_type)
+  {
+    case hardware_interface::HandleDataType::DOUBLE:
+      return "double";
+    case hardware_interface::HandleDataType::BOOL:
+      return "bool";
+    default:
+      return "unknown";
+  }
```

Comment:
isn't there something in the standard library for this?

Reply:
ah wait, sorry, that would be for the function doing the opposite operation.

Isn't this a to_string() method that should be in the HandleDataType or a friend of that class?

Reply:
I didn't know. Do you know any?
Can you share a reference

Reply:
Ahh yes, those are Better Enums. The standard Enums don't have that option. Maybe we can add it in a different PR

Reply:
I was checking earlier, unfortunately, the standard enums do not offer this functionality.

https://github.com/aantron/better-enums The only problem is this is BSD-2 License

Reply:
the `to_string()` suggestion still stands though ;)

Reply:
Ok, I'll get it done

Reply:
I just renamed the method to `to_string`. I cannot place this method in `hardware_info.hpp` as I'm getting multiple definitions issue. I hope this is OK

Reply:
@bmagyar I was able to do what you wanted :)